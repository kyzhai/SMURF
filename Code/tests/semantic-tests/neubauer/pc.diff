26a27,28
> 	ID: head h : t :: [a] -> a
> 	ID: tail h : t :: [a] -> [a]
