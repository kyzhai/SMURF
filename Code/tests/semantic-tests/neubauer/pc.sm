a = 1 %+ 4
b = 11 %+ 5

c = 9 %- 3
d = 1 %- 5
main = []
