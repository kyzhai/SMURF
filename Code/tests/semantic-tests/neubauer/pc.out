Program: 
	SVardef: 
		ID: a  :: Int

	1 %+ 4
	SVardef: 
		ID: b  :: Int

	11 %+ 5
	SVardef: 
		ID: c  :: Int

	9 %- 3
	SVardef: 
		ID: d  :: Int

	1 %- 5
	SMain: []
Global Scope: 
	ID: a  :: Int
	ID: b  :: Int
	ID: c  :: Int
	ID: d  :: Int
	ID: main  :: Unknown
	ID: print x :: a -> a
	ID: random  :: Int

